Shared/mods/deathmatch/logic/luadefs/CLuaElementDefsShared.cpp

```diff
@@ -50,6 +50,37 @@ int CLuaElementDefs::GetElementData(lua_State* luaVM)
     return 1;
 }
 
+int CLuaElementDefs::GetAllElementData(lua_State* luaVM)
+{
+    //  table getAllElementData ( element theElement )
+
+#ifdef MTA_CLIENT
+    CClientEntity* pElement;
+#else
+    CElement* pElement;
+#endif
+
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadUserData(pElement);
+
+    if (!argStream.HasErrors())
+    {
+        CLuaArguments Args;
+        CLuaArguments* pVariable = pElement->GetAllCustomData(&Args);
+        if (pVariable)
+        {
+            pVariable->PushAsTable(luaVM);
+            return 1;
+        }
+    }
+    else
+        m_pScriptDebugging->LogCustom(luaVM, argStream.GetFullErrorMessage());
```

Comment:
As mentioned in https://github.com/multitheftauto/mtasa-blue/pull/1612/files#r470167380, we no longer log warnings for usage mistakes. These are now errors! See #821.




```suggestion
        return luaL_error(luaVM, argStream.GetFullErrorMessage());
```